source/features/hide-markdown-diff.tsx

```diff
@@ -1,11 +1,19 @@
 import './hide-markdown-diff.css';
 import React from 'dom-chef';
+import select from 'select-dom';
 import delegate from 'delegate-it';
 import elementReady from 'element-ready';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
+// Fix missing indentation in changed quote blocks #4035
+function fixMissingIndentation(): void {
+	for (const changedBlockquote of select.all('.show-preview .changed > .changed_tag[data-before-tag="blockquote"]')) {
+		changedBlockquote.parentElement!.classList.add('ml-3');
+	}
```

Comment:
In the end I had to rely on some JS to fix the missing indentation issue.

At first I applied a margin on any siblings of `.changed_tag[data-before-tag="blockquote"]` but this causes issues with inline `<ins>` elements, so it has to be applied to the diff block wrapper.